oem/ibm/libpldmresponder/file_io_type_dump.cpp

```diff
@@ -531,18 +530,88 @@ int DumpHandler::read(uint32_t offset, uint32_t& length, Response& response,
     return readFile(resDumpDirPath, offset, length, response);
 }
 
-int DumpHandler::fileAckWithMetaData(uint32_t metaDataValue1,
+int DumpHandler::fileAckWithMetaData(uint8_t /*fileStatus*/,
+                                     uint32_t metaDataValue1,
                                      uint32_t metaDataValue2,
                                      uint32_t /*metaDataValue3*/,
                                      uint32_t /*metaDataValue4*/)
 {
     auto path = findDumpObjPath(fileHandle);
-    uint8_t fileStatus = (uint8_t)metaDataValue2;
+    uint8_t statusCode = (uint8_t)metaDataValue2;
     if (dumpType == PLDM_FILE_TYPE_RESOURCE_DUMP_PARMS)
     {
+        DBusMapping dbusMapping;
+        dbusMapping.objectPath =
+            resDumpEntryObjPath + std::to_string(fileHandle);
+        dbusMapping.interface = resDumpEntry;
+        dbusMapping.propertyName = "DumpRequestStatus";
+        dbusMapping.propertyType = "string";
+
+        pldm::utils::PropertyValue value =
+            "com.ibm.Dump.Entry.Resource.HostResponse.Success";
+
         std::cout << "fileAckWithMetaData with token: " << metaDataValue1
                   << " and status: " << metaDataValue2 << std::endl;
-        if (fileStatus != PLDM_SUCCESS)
+        if (statusCode == DumpRequestStatus::ResourceSelectorInvalid)
+        {
+            value =
+                "com.ibm.Dump.Entry.Resource.HostResponse.ResourceSelectorInvalid";
+        }
+        else if (statusCode == DumpRequestStatus::AcfFileInvalid)
+        {
+            value = "com.ibm.Dump.Entry.Resource.HostResponse.AcfFileInvalid";
+        }
+        else if (statusCode == DumpRequestStatus::PasswordInvalid)
+        {
+            value = "com.ibm.Dump.Entry.Resource.HostResponse.PasswordInvalid";
+        }
+        else if (statusCode == DumpRequestStatus::PermissionDenied)
+        {
+            value = "com.ibm.Dump.Entry.Resource.HostResponse.PermissionDenied";
+        }
+        else if (statusCode == PLDM_SUCCESS)
+        {
+            pldm::utils::PropertyValue value =
```

Comment:
Remove this as default value is set as success

Reply:
done